pepito_alonajane/index.html

```diff
@@ -0,0 +1,162 @@
+              placeholder="Your Name">
+             <textarea id="comment" 
+              placeholder="Your Comment"></textarea>
+             <button id="comment-btn"disabled>
+               Comment</button>
+          </form>
+        </div> 
+      </div>
+    <div class="container">
+      <div class="section">
+         <h2>Testimonials</h2>
+            <ul>
+              <li>Good work and creativity shows 
+               on her website.It also cute and
+               clearly shows her theme. - Sofia
+              </li>
+              <li>The consistent design elements 
+               and color schemes across the website
+               create a cohesive and professional
+               look. - Mikaella
+              </li>
+              <li>You have a knack for turning 
+               ordinary concepts into extraordinary 
+               works of art. -Precious 
+             </li>
+              <li>Kudos to you for creating this 
+               amazing webpage. Good job! - Eliza
+             </li>
+           </ul>
+         </div>  
+       </div>
+     <div class="container">
+       <div class="section">
+         <h2>My Thoughts About Conducting Online Class</h2>
+           <p>Conducting an online class during this high
+              heat index may cause a lot of problem or sticky 
+              situations such as having problem with internet 
+              connection or sickness amidst high heat index.
+              We and our seniors (BSIT 3-1), conducted face
+              to face classes might be because this mode of
+              learning will help us much through understand 
+              the lesson. I am bad at understanding things
+              and by that day,i wouldn't be able to understand 
+              a bit of information of javascript if i did not 
+              attend the face-to-face class.
+           </p>
+         </div>
+       </div>
+     <div class="image-container">
+       <div class="gallery">
+          <h2>Gallery</h2>
+             <img src="https://i.pinimg.com/736x/3a/58/1d/3a581d415c37c42b793cc7914ce8118c.jpg">
+             <img src="https://i.pinimg.com/736x/3a/a0/38/3aa038ac501f02ac6b3c0f06b905cb6c.jpg">
+             <img src="https://i.pinimg.com/736x/9c/eb/36/9ceb3620d92fc7048a6b8188f84c7294.jpg">
+             <img src="https://i.pinimg.com/736x/90/57/40/905740a85b432913dfbe9906ac39a44f.jpg">
+             <img src="https://i.pinimg.com/736x/cb/9c/5b/cb9c5be444873f9054e1f9a0c99616bc.jpg">
+       </div>
+     </div>
+   <div class="contact">
+     <div class="header">
+        <h2> Contact Me</h2>
+     </div>
+         <form>
+            <div class="email">
+               <label for="email">Email:</label>
+               <input type="email" id="email" name="email">
+             </div>
+            <div class="full-name">
+               <label for="full-name">Full Name:</label>
+               <input type="text" id="full-name" name="full-name">
```

Comment:
(same error)
id is should be written in snake case.
`<input type="text" id="full_name" name="full-name">`